e107_languages/English/admin/lan_admin.php

```diff
@@ -328,7 +328,7 @@
 define("LAN_VIEW_ALL","View All");
 
 define("LAN_BAN","Ban");
-
+define("LAN_PATH", "Path");
```

Comment:
Unless 'path' is used often throughout the admin of e107, (I don't think it is) this should be in lan_plugin.php 
What do you think @MikeyGMT ?